packages/manager/modules/pci/src/projects/new/config/controller.js

```diff
@@ -18,26 +19,61 @@ export default class PciProjectNewConfigCtrl {
     }).join(', ');
   }
 
-  /* -----  End of Helpers  ------ */
-
-  /* =============================
-  =            Events            =
-  ============================== */
-
-  onProjectConfigFormSubmit() {
-    let setDescrPromise = Promise.resolve(true);
-
+  setCartProjectItem() {
     if (
       this.model.description &&
       !this.cart.projectItem.descriptionConfiguration
     ) {
-      setDescrPromise = this.pciProjectNew.setCartProjectItemDescription(
+      return this.pciProjectNew.setCartProjectItemDescription(
         this.cart,
         this.model.description,
       );
     }
 
-    return setDescrPromise.then(() => this.goToPayment());
+    return Promise.resolve();
```

Comment:
As we are in a angularJS controller, we better return 
```suggestion
    return this.$q.when();
```
as Promise are not synced with `$scope` digest cycles

Reply:
fixed